pages/studentPortal/web/React/tailwindcss-colourstyling.tsx

```diff
@@ -0,0 +1,70 @@
+import { useRouter } from "next/router";
+import React from "react";
+import LessonComponent, {
+  LessonComponentData,
+} from "../../../../components/coding/studentPortal/LessonComponent";
+import { Button } from "../../../../components/ui/Button";
+
+const TailwindColourStyling = ({ lessonComponents }) => {
+  const router = useRouter();
+
+  const handleContinue = () => {
+    router.push("/studentPortal/web/React/tailwindcss-gridflex");
+  };
+  return (
+    <>
+      <div className="grid grid-cols-1 gap-8 px-4 pt-4 m-8 sm:px-12">
+        {lessonComponents.map((it, index) => (
+          <LessonComponent data={it} key={index} />
+        ))}
+      </div>
+      <div className="flex my-8 mr-8 sm:justify-end">
+        <Button onClick={handleContinue} label="Continue" />
+      </div>
+    </>
+  );
+};
+
+export async function getServerSideProps({ params }) {
+  const lessonComponents: LessonComponentData[] = [
+    {
+      component: "title",
+      text: "Tailwind Lesson 2: Background Color & Font Styling",
+    },
+
+    {
+      component: "description",
+      text: "TailwindCSS can add colour to the backgroud of a page, and can modify the text colour, size, and emphasis. ",
+    },
+    {
+      component: "code-sandbox",
+      title: "Background Colour Example",
+      link: "https://codesandbox.io/embed/backgroundstyling-02elf9?fontsize=14&hidenavigation=1&theme=dark",
+    },
+    {
+      component: "description",
+      text: "Notice that on line 5, bg-blue-500 is used above to style the background colour of the webpage. In general, bg-colour-numberbetween0and900 is the syntax used to style the background, where the final number represents the shade value for the chosen color. Additionally, you can use rgb(x,y,z) values to specify the exact colour of the background.",
```

Comment:
nit but i think it's clearer if you use brackets to denote where a value goes i.e bg-[colour]-[0-900]

might also be worth mentioning that the significance of lower vs. higher numbers i.e a lower number is a lighter variation of the background color